src/test/java/org/openmrs/module/mirebalais/smoke/helper/SmokeTestDriver.java

```diff
@@ -26,17 +30,23 @@ public SmokeTestDriver() {
 
     }
 
-    private static void setupChromeDriver() {
+    private void setupChromeDriver() {
         URL resource = null;
         ClassLoader classLoader = BasicMirebalaisSmokeTest.class.getClassLoader();
 
-        if(SystemUtils.IS_OS_MAC_OSX) {
-            resource = classLoader.getResource("chromedriver/mac/chromedriver");
-        } else if(SystemUtils.IS_OS_LINUX) {
-            resource = classLoader.getResource("chromedriver/linux/chromedriver");
-        } else if(SystemUtils.IS_OS_WINDOWS) {
-            resource = classLoader.getResource("chromedriver/windows/chromedriver.exe");
+        if (StringUtils.isBlank(properties.getChromeDriverExecutable())) {
+            if(SystemUtils.IS_OS_MAC_OSX) {
+                resource = classLoader.getResource("chromedriver/mac/chromedriver");
+            } else if(SystemUtils.IS_OS_LINUX) {
+                resource = classLoader.getResource("chromedriver/linux/chromedriver");
+            } else if(SystemUtils.IS_OS_WINDOWS) {
+                resource = classLoader.getResource("chromedriver/windows/chromedriver.exe");
+            }
         }
+        else {
+            resource = classLoader.getResource(properties.getChromeDriverExecutable());
```

Comment:
Looks good to me - feel free to merge as-is.

It would be great if this could either load from the file system or the class path, do you know if this is how this works?  Or are you expecting to need to add these to a classpath folder as needed?

Reply:
Yeah, this is was pretty quick, but, yeah, we should probably update allow loading via file system.